package/amazon-alexa/avs-device-sdk/0002-Synchronise_rapidjson_version_with_smart_screen.patch

```diff
@@ -1,94 +0,0 @@
-diff --git a/ThirdParty/rapidjson/rapidjson-1.1.0/include/rapidjson/internal/biginteger.h b/ThirdParty/rapidjson/rapidjson-1.1.0/include/rapidjson/internal/biginteger.h
```

Comment:
Does alexa-device-sdk and the smart screen uses now the same version of rapidjson? Is rapid json coming from the buildroot or are there still any internal downloaders in some pacakges?

Reply:
rapidjson coming from Buildroot now